spreadsheet_exporter.gemspec

```diff
@@ -17,5 +17,6 @@ Gem::Specification.new do |s|
   s.files = Dir["{app,config,db,lib}/**/*", "MIT-LICENSE", "Rakefile", "README.md"]
   s.test_files = Dir["test/**/*"]
 
+  s.add_dependency "activesupport", ">= 6"
```

Comment:
Give us access to `to_json`, `deep_stringify_keys` and whatever hash key handlers we need.